docs/source/tutorials/uncertainty.rst

```diff
@@ -0,0 +1,148 @@
+    Inference - Iteration 11: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.85s/it]
+    Inference - Iteration 12: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.92s/it]
+    Inference - Iteration 13: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.83s/it]
+    Inference - Iteration 14: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.84s/it]
+    Inference - Iteration 15: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.87s/it]
+    Inference - Iteration 16: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.81s/it]
+    Inference - Iteration 17: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.95s/it]
+    Inference - Iteration 18: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:09<00:00,  1.82s/it]
+    Inference - Iteration 19: 100%|██████████████████████████████████████████████████████████████████████████████████| 5/5 [00:08<00:00,  1.71s/it]
+
+The Monte Carlo samples are then used to compute uncertainties measures for each image. The results are saved under
+``pred_masks``.
+
+.. code-block:: bash
+
+    Uncertainty Computation: 100%|███████████████████████████████████████████████████████████████████████████████████| 5/5 [01:31<00:00, 18.28s/it]
+
+Six files are generated during this process for each testing image:
+- ``*_soft.nii.gz``: Soft segmentation (i.e. values between 0 and 1) which is generated by averaging the Monte Carlo
+samples.
+- ``*_pred.nii.gz``: Binary segmentation obtained by thresholding ``*_soft.nii.gz`` with ``1 / (Number of Monte Carlo
+iterations)``.
+- ``*_unc-vox.nii.gz``: Voxel-wise measure of uncertainty derived from the entropy of the Monte Carlo samples. The
+higher a given voxel value is, the more uncertain is the prediction for this voxel.
+- ``*_unc-avgUnc.nii.gz``: Structuve-wise measure of uncertainty derived from the mean value of ``*_unc-vox.nii.gz``
+within a given connected object (e.g. a lesion, grey matter).
+- ``*_unc-cv.nii.gz``: Structuve-wise measure of uncertainty derived from the coefficient of variation of the volume
+of a given connected object across the Monte Carlo samples. The higher a given voxel value is, the more uncertain is the
+ prediction for this voxel.
+- ``*_unc-iou.nii.gz``: Structuve-wise measure of uncertainty derived from the Intersection-over-Union of the
+predictions of a given connected object across the Monte Carlo samples. The lower a given voxel value is, the more
+uncertain is the prediction for this voxel.
+
+These files can further be used for post-processing to refine the segmentation.
```

Comment:
> Another side note not directly related to your tutorial: When on the Technical Features page, we can't see the 2 new tutorials in the side bar. On the ivadomed.org website the issue doesn't seem to be there, so it might just be on my local build

i've added the branch build on RTD, and added the link in https://github.com/ivadomed/ivadomed/pull/399#issue-468759511

Reply:
@andreanne-lemay thanks. Attempt to fix links and bullet points [here](https://github.com/ivadomed/ivadomed/pull/399/commits/aefce8b272ebfb76803c4eefdcaa7592f7fe6e83) and [here](https://github.com/ivadomed/ivadomed/pull/399/commits/89fe8e03dc62573730d5c962c413805ca86d9781), respectively.